src/lib/validations/sql.js

```diff
@@ -247,24 +247,29 @@ const perLineValidations = ( line: string ) => {
 	lineNum += 1;
 };
 
+const execute = ( line: string, isImport: boolean = true ) => {
+	perLineValidations( line, isImport );
+};
+
+const postLineExecutionProcessing = async ( { fileName, isImport }: PostLineExecutionProcessingParams ) => {
```

Comment:
Do we need this function any more? It's a 1:1 proxy with `postValidation` as far as I can see.

Reply:
Ah yes, removed!

Reply:
It looks like we actually need this to pass down `isImport`. Reverted in #639